functions/src/import-csv.ts

```diff
@@ -76,24 +79,26 @@ async function importCsv(req, res) {
     await Promise.all(inserts);
     const count = inserts.length;
     console.log(`Inserted ${count} rows`);
-    await res.status(HttpStatus.OK).end(JSON.stringify({ count }));
+    res.status(HttpStatus.OK).end(JSON.stringify({ count }));
   });
 
   busboy.on("error", (err) => {
-    console.err("Busboy error", err);
-    next(err);
+    console.error("Busboy error", err);
+    // TODO bring next function back ????
+    // next(err);
```

Comment:
I don't think `next` is accessible here unless we pass an Express app to onHttpRequest. Instead, I think we can be just unpipe and return an error as we do above at line 69-70.

Reply:
like this?
```
busboy.on("error", (err) => {
        console.err("Busboy error", err);
        res.unpipe(busboy);
});
```

Reply:
Yes, and we may want to also set the response status to 500 and include the error message there as well.